tests/test_utils.py

```diff
@@ -1,4 +1,26 @@
+from binascii import unhexlify
+
 # Default variables used for the ntlm_context
 username = 'username'
 domain = 'domain'
 password = 'password'
+
+timestamp = unhexlify('0000000000000000')
+client_challenge = unhexlify('aaaaaaaaaaaaaaaa')
+session_base_key = unhexlify('55555555555555555555555555555555')
```

Comment:
This was the easiest way to create a bytes object in Python that is supported from Python 2.6 to 3.5. If I didn't have to worry about 2.6 I could probably do `bytes('\x00\x00\x00\x00\x00\x00\x00\x00', 'utf-8')` instead unless there is a better way?